lib/stripe/util.rb

```diff
@@ -258,17 +323,64 @@ def self.check_array_of_maps_start_keys!(arr)
         end
       end
     end
+    private_class_method :check_array_of_maps_start_keys!
 
-    # Constant time string comparison to prevent timing attacks
-    # Code borrowed from ActiveSupport
-    def self.secure_compare(a, b)
-      return false unless a.bytesize == b.bytesize
+    # Uses an ANSI escape code to colorize text if it's going to be sent to a
+    # TTY.
+    def self.colorize(val, color, isatty)
+      return val unless isatty
 
-      l = a.unpack "C#{a.bytesize}"
+      mode = 0 # default
+      foreground = 30 + COLOR_CODES.fetch(color)
+      background = 40 + COLOR_CODES.fetch(:default)
 
-      res = 0
-      b.each_byte { |byte| res |= byte ^ l.shift }
-      res == 0
+      "\033[#{mode};#{foreground};#{background}m#{val}\033[0m"
+    end
+    private_class_method :colorize
```

Comment:
Just a quick note: yeah, I implemented my own terminal colorization because it's fairly simple to do, extremely stable (we can safely expect this to never change), and it avoids bringing in a new dependency.